### ISSUE TITLE
Navigation bar gradient

### ISSUE THREAD

**Issue by Samq64** [ScratchAddons/ScratchAddons#6022]
### Why this would be helpful

It looks pretty cool.

### My suggestion

An addon that adds a gradient to the navigation bar.

### Possible alternatives

Making it an option in `dark-www` or `discuss-button`.

### Additional context

Closed PR: #6010

**Comment by WorldLanguages** [comment]
Worth watching: https://www.youtube.com/watch?v=Uh95jZPTDfw

**Comment by mxmou** [comment]
I don't think this is a bad idea. Instead of a new addon or setting, the color setting type could be extended to allow choosing a solid color or a gradient.

**Comment by Secret-chest** [comment]
> I don't think this is a bad idea. Instead of a new addon or setting, the color setting type could be extended to allow choosing a solid color or a gradient.

> I don't think this is a bad idea. Instead of a new addon or setting, the color setting type could be extended to allow choosing a solid color or a gradient.

What if we need it for text? Do we have a separate gradient/solid setting?

**Comment by Samq64** [comment]
> What if we need it for text? Do we have a separate gradient/solid setting?

It would probably be opt-in like transparency.
